src/pages/UserHome.jsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+import Box from '@material-ui/core/Box';
```

Comment:
Tip: instead of this you can also import `Box` in the long list of imports from `@material-ui/core`.